app/controllers/stripe-setup/vat-number-company-number/vat-number/get.controller.js

```diff
@@ -0,0 +1,17 @@
+'use strict'
+
+// NPM Dependencies
+const lodash = require('lodash')
+
+// Local dependencies
+const response = require('../../../../utils/response')
+
+module.exports = (req, res) => {
+  // initialise pageData
+  let pageData = {
+    errors: lodash.get(req, 'session.stripeSetup.vatNumber.errors'),
+    vatNumber: lodash.get(req, 'session.stripeSetup.vatNumber.vatNumber', '')
```

Comment:
If we have session variables set, do we need to make sure we delete them after we get them?
A scenario could be that a user fills in the page for one service, then tries to fill it in for another service afterwards and finds there are pre-filled fields.